lib/settings/element/user_info_section.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+Widget _shortUserInfo(BuildContext context) {
+  return Container(
+    width: MediaQuery.of(context).size.width,
+    height: ((MediaQuery.of(context).orientation == Orientation.landscape)

```

Comment:
@damoklov as an improvement, you can do something like this in the beginning of your method:
```
final screenWidth = MediaQuery.of(context).size.width;
final screenHeight = ...;
final orientation = ....; 
```

and then use those variables instead of using `MediaQuery.of()` each time